UPGRADE-4.x.md

```diff
@@ -1,12 +1,26 @@
 UPGRADE 4.x
 ===========
 
+UPGRADE FROM 4.x to 4.x
+========================
+
+### SonataEasyExtends is deprecated
+
+Registering `SonataEasyExtendsBundle` bundle is deprecated, it SHOULD NOT be registered.
+Register `SonataDoctrineBundle` bundle instead.
+
+UPGRADE FROM 4.2.3 to 4.3
+=========================
+
 ### Deprecated `TwoStepVerificationCommand` with no arguments
 
 Attempting to run that command with 2FA enabled and without building the
 command with its `$helper` and `$userManager` arguments beforehand is
 deprecated. Make sure you provide both arguments.
 
+UPGRADE FROM 4.1 to 4.1.1
+=========================
```

Comment:
I forgot to do my job… it's weird that the heading mentions a patch version though, we are not supposed to introduce deprecations in patch versions…

Reply:
And it is another PR of mine, I think I introduced a deprecation on a patch, and without the proper changelog, because I could not find one for that release